content/api/authentication/systemuserapi/clientdelegation/_index.nb.md

```
@@ -211,32 +246,40 @@ Den unike identifikatoren til klienten som skal legges til systembrukeren
 
 #### client
+
 Den unike identifikatoren til klienten som skal fjernes fra systembrukeren
 
 ### Eksempel på forespørsel
+
 {{environmenturl}}/authentication/api/v1/enduser/systemuser/clients/?agent=58cd5a57-ea49-4d04-bf7d-d48b338c68db&client=ff254c60-d02a-4ae8-bcd1-34cce38a823a
 
 ### Eksempel på respons
+
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)